torch/_functorch/aot_autograd.py

```diff
@@ -948,6 +952,11 @@ def aot_module_simplified(
                 seen_sources.add(source)
                 aot_autograd_arg_pos_to_source.append(source)
 
+                if "tensor_dict" in node.meta and node.meta["tensor_dict"].get(
+                    "_dynamo_static_input_type", None
```

Comment:
i didn't now about `meta['tensor_dict']`, does dynamo populate it for every fx node in the dynamo graph?

Reply:
ah, is the idea that dynamo calls `mark_static_address()` on all params/buffers that it encounters?

Reply:
Yeah, I also didn't know either, but dynamo does populate it for all nodes that have a real tensor input (I think just placeholders, but that's all I need for this anyway)

Reply:
And yes to your second question, dynamo will call that on all params and buffers within an NN module and parameters it encounters outside of it.